source/element_parameters.md

```diff
@@ -0,0 +1,225 @@
+```
+
+Default values for the limits are `[-Inf, Inf]` for both `x_limit` and `y_limit`.
+
+### misalignment_moves_aperture
+
+The `misalignment_moves_aperture` parameter determines whether misaligning an element 
+affects the placement of the aperture. The default is `False`. 
+A common case where `misalignment_moves_aperture` would be `False` is when a beam pipe,
+which incorporates the aperture, is not physically touching the surrounding magnet element. 
+When tracking a particle, assuming that there are only apertures at the element ends, 
+the order of computation with `misalignment_moves_aperture` set to `False` could be
+```{code} yaml
+  1) Start at upstream end of element
+  2) Check upstream aperture if there is one.
+  3) Convert from branch coordinates to body coordinates.
+  4) Track through the element body.
+  5) Convert from body coordinates to branch coordinates.
+  6) Check downstream aperture if there is one.
+  7) End at downstream end of element.
+```
+With `misalignment_moves_aperture` set to `True`, the computation order could be
+```{code} YAML
+  1) Start at upstream end of element
+  2) Convert from branch coordinates to body coordinates.
+  3) Check upstream aperture if there is one.
+  4) Track through the element body.
+  5) Check downstream aperture if there is one.
+  6) Convert from body coordinates to branch coordinates.
+  7) End at downstream end of element.
+```
+
+### material
+
+The `material` parameter sets the material of the aperture. 
+Using chemical formulas like `Cu` and `Fe` are the most portable.
+
+### wall2d
+
+The `WALL2D` setting for `aperture_shape` is for defining an aperture using a 
+set of vertex points as illustrated in {numref}`f:aperture`B. 
+Between vertex points, the aperture can can follow a straight line or the arc of an ellipse. 
+The vertex points are specified by setting the `wall2d` parameter. This parameter has three
+subcomponents
+```{code} yaml
+center              # (x, y) center pointer.
+absolute_vertices   # Boolian. Default is False.
+vertices            # Ordered list of vertex points.
+```
+Example:
+```{code} yaml
+Aperture:
+  wall2d:
+    center: [-0.045, 0.011]
+    absolute_vertices: True
+    vertices:
+      - point: [0.023, 0.069]      # V1
+      - point: [-0.025, 0.067]     # V2
+      - radius: [0.08, 0.04]
+      - tilt: 0.12
+      - point: [-0.088, 0.036]     # V3
+      - point: [-0.088, -0.021]    # V4
+      - point: [0.023, -0.033]     # V5
+```
+This corresponds roughly to what is shown in {numref}`f:aperture`.
+
+If the boolean `absolute_vertices` is set `False`, which is the default,
+the vertex point positions are with respect to the `center` point. 
+That is, the vertex point positions in absolute terms are the positions given with each vertex plus
+the position of the `center`. If `absolute_vertices` is `True`, the vertex point positions 
+are independent of the `center` point. The default position of the `center` is the origin.
```

Comment:
I'm not sure I understand the purpose of this flag.  When would a user need the case `absolute_vertices = True`?  Isn't this just equivalent to just using `absolute_vertices = False` together with the default `center = [0,0]`?

Reply:
Yes. If the center = [0,0] then the setting of absolute_vertices does not matter. absolute_vertices is just a convenience and not a necessity if center is not equal to [0,0].

Reply:
I think adding this boolean input adds an unnecessary complication, but that's just one opinion.

Reply:
The advantage of a`bsolute_vertices` = True is that it is easier to visualize where the aperture is (since the aperture position is independent of `center`). The advantage of `absolute_vertices` = False is that moving the aperture as a whole is easy (just vary the `center`).  Which one would you give up if `absolute_vertices` was not present?

Note: If the aperture does not enclose the origin (0,0) then `center` (which must be inside the aperture) cannot be (0,0).